notebooks/native_graph_kernels.ipynb

```diff
@@ -0,0 +1,313 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "38c7d14c",
+   "metadata": {},
+   "source": [
+    "# GP Regression on Molecules #\n",
+    "\n",
```

Comment:
Should we change the text in the notebook?

Reply:
Will do